tests/unit/update-manager-test.js

```diff
@@ -198,4 +197,53 @@ describe('UpdateManager', () => {
 
   })
 
+  describe('editable', () => {
+    const self = {err: ''}
+    let updater, docuri, rterm, options, userCallback, loadStub
+    var loadStatus = 200
+
+    beforeEach(() => {
+      options = {}
+      userCallback = () => {}
+
+      updater = new UpdateManager()
+      // updater.store.fetcher.webOperation = sinon.stub().resolves({ ok: true, status: 200, statusText: "Dummy stub 1"})
```

Comment:
This needs to be included for the last test to work, since the last test will fetch data